pytravharv/__main__.py

```diff
@@ -45,119 +43,49 @@ def get_arg_parser():
         "-n",
         "--name",
         type=str,
+        required=False,
         default=None,
         help="Name of the configuration to use",
     )
 
+    parser.add_argument(
+        "-m",
+        "--mode",
+        choices=["memory", "uristore"],
+        default="memory",
+        required=True,
+        help="Mode to use, either memory or uristore. Default is memory. If memory is used, the target store will be a temporary in-memory store. If uristore is used, the target store will be a URI store.",
+    )
+
     parser.add_argument(
         "-o",
-        "--output-folder",
+        "--output",
         type=str,
-        default=os.path.join(os.getcwd(), "output"),
-        help="Folder to output files to",
+        default=None,
+        required=False,
+        help="File to write output to, if not specified, output will be written to stdout",
+    )
+
+    parser.add_argument(
+        "-c",
```

Comment:
not clear what this does, and why it is needed?

Reply:
reading further I get the impression that this is about pre-loading the store with content to get started (e.g. making sure there are actual subject triples to be found in sparql statements)

similar to the `-d`, `--dump` (now `--output`) this is not a concern of travharv strictly imho

but I see how it can be useful in this cli context 

suggestion: `-l`, `--load`  gives a better indication of the fact that this is loading some initial-state into the store to be used ?